lisa/wa.py

```diff
@@ -255,6 +255,13 @@ def _needs_params(col):
             for name, param in parameters[1:]
         )
 
+    @property
+    @memoized
+    def jobs(self):
+        outputs = list(discover_wa_outputs(self.path))
+        if len(outputs) > 1:
+            raise DeprecationWarning('Passing a directory containing multiple outputs of "wa run" to WAOutput is deprecated. Please, create one object per run.')
```

Comment:
use this: https://docs.python.org/3/library/warnings.html#warnings.warn instead of raising an exception